### DISCUSSION TITLE
Translation memory - deletion limited to 999 entities at a time.

### DISCUSSION THREAD

**Discussion by ianathompson** [WeblateOrg/weblate#4740] [General] [answered]
When trying to delete the translation memory from the Admin portal, you can only delete 999 entities at a time. 1000 and over causes Bad Request. Feature or bug?
I don't have to time to do the issue if it is.
_Running on 4.2.2 docker_

**Comment by nijel** [comment, marked as answer]
It's caused by Django request limits, see https://docs.djangoproject.com/en/3.1/ref/settings/#data-upload-max-number-fields

You can bulk delete translation memory entries on project/user/site memory management pages, but the filtering is really limited there.

**Reply by ianathompson** [reply]
Thanks. Will adjust it and give it a try.
